src/Components/LovedTracks.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react';
+import LovedTrackCard from "./LovedTrackCard";
+
+function LovedTracks() {
+
+    const [result,setResult] = useState([]);
+
+    useEffect(() => {
+        setLovedTracks()
+    }, []);
+
+    async function getLovedTracks() {
+        try {
+            const response = await fetch(`https://ws.audioscrobbler.com/2.0/?api_key=906db58ae0258689ba249d53210358ee&method=user.getlovedtracks&user=Foxyb0y&format=json`, {
+                method: 'GET'
+            });
+            return await response.json()
+        } catch (err:any) {
+            alert(`Произошла ошибка поиска${err.message}`);
```

Comment:
стоит пробел добавить
```suggestion
            alert(`Произошла ошибка поиска ${err.message}`);
```